camera/texture-view/build.gradle

```diff
@@ -12,7 +12,7 @@ android {
         }
         externalNativeBuild {
             cmake {
-                arguments '-DANDROID_STL=gnustl_static', '-DANDROID_TOOLCHAIN=clang'
+                arguments '-DANDROID_STL=c++_static', '-DANDROID_TOOLCHAIN=clang'
```

Comment:
Why are these set explicitly? Just let the cmake toolchain decide what's best for the given NDK. The only time you should need to set this explicitly is if you need the shared STL rather than the static STL, or if you're using a feature that is not available in gnustl (but that by definition is not the case for any of these, since they were already gnustl).

Same goes for `ANDROID_TOOLCHAIN`. Clang has been the default for ages.

Reply:
when setting explicitly, application get what required; not to worry from version to version default. In early days, a couple of samples have to be built with C++_* and clang ( gnustl_ won't build ) at that point to align all together, most settings are explicit.